x-pack/plugins/monitoring/server/es_client/__tests__/instantiate_client.js

```diff
@@ -7,7 +7,7 @@
 import expect from '@kbn/expect';
 import sinon from 'sinon';
 import { noop } from 'lodash';
-import { exposeClient, hasMonitoringCluster } from '../instantiate_client';
+import { instantiateClient, hasMonitoringCluster } from '../instantiate_client';
```

Comment:
is `fixture_read_file.txt` used anywhere?

Reply:
I'm not following? What file is that?

Reply:
https://github.com/elastic/kibana/blob/ee8086b75af98391935977e2bb2f908c366dd8ff/x-pack/plugins/monitoring/server/es_client/__tests__/fixture_read_file.txt